docs/plugins/harbor.md

```diff
@@ -68,4 +68,15 @@ This plugin support `Ingress`, `ClusterIP`, `NodePort`, `LoadBalancer` , You can
 --8<-- "harbor.yaml"
 ```
 
-Currently, except for `values_yaml`, all the parameters in the example above are mandatory.
+#### Default Configs
+
+| key              | default value            | description                                |
+| ----             | ----                     | ----                                       |
+| chart.chart_name | harbor/harbor            | chart name                                 |
+| chart.timeout    | 10m                      | this config will wait 10 minutes to deploy |
```

Comment:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/36830265/183551049-c1b5e6e7-018a-4d94-9891-a797b2285198.png">

Please edit this comment.